Dockerfile

```diff
@@ -11,6 +13,8 @@ COPY --from=golang /go/ /go/
 COPY --from=golang /usr/local/go/ /usr/local/go/
 COPY --from=tfdocs /usr/local/bin/terraform-docs /usr/local/bin/
 COPY --from=terraform /bin/terraform /usr/local/bin
+COPY --from=shellcheck /bin/shellcheck /usr/local/bin
+COPY --from=bats /opt/bats /usr/local/bin
```

Comment:
I don't think this is working quite right. You can see in the CI job that it is [still installing bats](https://travis-ci.com/github/plus3it/tardigrade-ci/jobs/496553728#L576-L812) as part of the `bats/test` run. That is a sign that there is no `bats` executable in the path.

If we look at their Dockerfile, we see these two lines: 

https://github.com/bats-core/bats-core/blob/0b31b02f46f15223df8858ef77a193e9e3c6c58e/Dockerfile#L10-L11

The first one is creating a symlink to the `bats` executable, the second one is putting the files into the `/opt/bats` directory. We can setup our container the same way:

```
COPY --from=bats /opts/bats /opts/bats
COPY . /${PROJECT_NAME}
RUN apt-get update -y && apt-get install -y \
    xz-utils \
    curl \
    jq \
    unzip \
    make \
    && ln -s /opt/bats/bin/bats /usr/local/bin/bats \
    && make -C /${PROJECT_NAME} install \
    && touch /.dockerenv \
    && rm -rf /var/lib/apt/lists/*
```

Reply:
Ah okay. Yeah I saw that and just thought if we copy from the most recent directory manipulated in the Dockerfile it would be enough :/ I'll make the change!

Reply:
You need both the COPY line and the RUN line from their Dockerfile... In my comment, I combined their RUN line with ours...

```
    && ln -s /opt/bats/bin/bats /usr/local/bin/bats \
```